app/model/exchange/EventAllocations.scala

```diff
@@ -27,9 +28,12 @@ object EventAssessorAllocationsSummaryPerSkill {
   implicit val format = Json.format[EventAssessorAllocationsSummaryPerSkill]
 }
 
-case class EventWithAllocationsSummary(event: Event,
-                                       confirmedCandidates: Int,
-                                       assessorAllocations: List[EventAssessorAllocationsSummaryPerSkill])
+case class EventWithAllocationsSummary(
+  date: LocalDate,
```

Comment:
I do not think it's needed. You can always access date from event. (see my comment at frontend PR)